package.json

```diff
@@ -9,7 +9,6 @@
         "url": "git@github.com:box/box-content-preview.git"
     },
     "devDependencies": {
-        "@babel/core": "^7.18.6",
```

Comment:
Do we not still need this for Preview's build?

Reply:
Good catch, I thought I added this when I was testing out babel CLI to help minify the file. I'll add back.